### PR TITLE
Feat: ECS on EC2

### LINKED ISSUE #79: Add support for ECS with EC2 launch type
Infracost currently supports ECS with Fargate launch type, but not EC2 launch type.

### PR DESCRIPTION
**Pull request by sinabakh**
Fix #79

### CODE REVIEW

File: infracost-usage-example.yml [hunk 1]
```diff
@@ -158,6 +158,9 @@ resource_usage:
     monthly_log_lines: 5000000          # Monthly number of real-time log lines.
     custom_ssl_certificates: 3          # Number of dedicated IP custom SSL certificates.
 
+  aws_autoscaling_group.my_asg:
+    instances_count: 15 # Number of instances in the autoscaling group.
```
Comment (aliscott) [inline infracost-usage-example.yml:162]: We use `instances` elsewhere and in the contributing docs (https://github.com/infracost/infracost/blob/master/CONTRIBUTING.md#usage-file-notes), so I think we should keep this consistent.

**Review by aliscott — APPROVED** [review]
Looks good! Thanks @sinabakh.

File: infracost-usage-example.yml (hunk 1 above)
Comment (sinabakh) [reply infracost-usage-example.yml:162]: Unfortunately, I didn't exactly get your suggestion. I believe `count` itself does not make much sense in this use case.

Comment (aliscott) [reply infracost-usage-example.yml:162]: `instances_count` -> `instances`

We use `instances` elsewhere as well.

Comment (sinabakh) [reply infracost-usage-example.yml:162]: Ahh sorry, my bad. Will fix it.

**Pull request merged**